CHANGELOG.md

```diff
@@ -375,7 +375,7 @@ FEATURES
     - [x/gov] Can now query governance proposals by ProposalStatus
 
 * Gaia CLI  (`gaiacli`)
-    - [x/gov] added `query-proposals` command. Can filter by `depositer`, `voter`, and `status`
```

Comment:
Can you revert the changelog change? We'd like the history here to be as immutable as possible :)

Reply:
Applied